src/app/donation-start/donation-start.component.ts

```diff
@@ -608,13 +608,8 @@ export class DonationStartComponent implements AfterContentChecked, AfterContent
         this.stripeError = `Payment processing failed: ${result.error.message}`;
         this.stripeResponseErrorCode = result.error.code;
         if (this.isBillingPostCodeInvalid()) {
-          console.log(this.paymentGroup);
+          this.paymentGroup.controls.billingPostcode.setValidators([ValidateBillingPostCode]);
```

Comment:
This is going to remove the required and pattern validators which are added when in Stripe credit card mode as it sets the full array to just one.

Note that these must remain dynamic because the field is not applicable when using Payment Request Buttons so the validators are sometimes set back to `[]`.

Similarly, we must have logic which removes this validator again when the postcode or card number is changed, otherwise it will continue to block submitting the form even after a correction.